gigascience/gigadb-website
# Fix outstanding issues in dataset page

# Pull request for issue: #1525

This is a pull request for the following functionalities:

* Fix outstanding accessibility issues in http://gigadb.gigasciencejournal.com:9170/dataset/100245 pages

## How to test?

* Navigate to different dataset pages, e.g.
  * http://gigadb.gigasciencejournal.com:9170/dataset/100245
  * http://gigadb.gigasciencejournal.com:9170/dataset/100006
  * http://gigadb.gigasciencejournal.com:9170/dataset/100056
  * http://gigadb.gigasciencejournal.com:9170/dataset/100245
* Run WAVE a11y audit and confirm that there are no "errors" reported
* Inspect the page appearance and check that nothing looks "strange" (hard refresh the page to prevent the browser from using cached CSS)
* Navigate page with keyboard and confirm that all interactive elements are focusable

## How have functionalities been implemented?

A number of features were added:

* Fixed heading hierarchy
* Added focus state (and made keyboard focusable) some elements that needed it, e.g. Previous and Next buttons, Cite Dataset dropdown
![image](https://github.com/gigascience/gigadb-website/assets/143437854/7a724c6c-d6b0-4c8b-aec1-157d25fbbebf)
* Made the DOI badge non-focusable since it is non-interactive (unclear why it was a link)
![image](https://github.com/gigascience/gigadb-website/assets/143437854/21b86bf2-3406-463c-bb9e-b42bea6a779b)
* Add a message above the table, indicating that it can be sorted
* Branded styles to modal buttons
* Turned peer review publicaitons into a list
* Updated style and logic of pagination buttons
![image](https://github.com/gigascience/gigadb-website/assets/143437854/eb83f4f0-ba03-4722-b6eb-ab004497db9f)


## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
